packages/manager/apps/web/client/app/hosting/user-logs/add/hosting-user-logs-add.controller.js

```diff
@@ -89,19 +89,30 @@ angular.module('App').controller(
         type: 'action',
       });
       this.$scope.resetAction();
-      return this.Hosting.userLogsCreate(
+
+      this.Hosting.getOwnLogs(
+        this.$stateParams.productId,
         this.$stateParams.productId,
```

Comment:
Can't we use just one parameter ?

Reply:
the api to get ownLog takes 2 parameters : the serviceName (to get ownLogs for the hosting) and a filter to get ownLog on a particular domain name of the hosting. Here we want the default own log of the hosting, so we filter on the main domain name which is the service name.